include/dlaf/init.h

```diff
@@ -38,7 +38,15 @@ struct configuration {
   std::size_t num_np_gpu_streams_per_thread = 3;
   std::size_t num_hp_gpu_streams_per_thread = 3;
   std::size_t umpire_host_memory_pool_initial_bytes = 1 << 30;
+  std::size_t umpire_host_memory_pool_next_bytes = 1 << 30;
```

Comment:
If I were to rename these I'd probably do:
```suggestion
  std::size_t umpire_host_memory_pool_initial_block_bytes = 1 << 30;
  std::size_t umpire_host_memory_pool_next_block_bytes = 1 << 30;
```
Is that an improvement? Other ideas?

Reply:
I've updated to the suggestion above.